content/de/pressreview/2014_10_18_20min.md

```diff
@@ -0,0 +1,6 @@
+Title: Was macht die DDR-Flagge im Überwachungs-Büro?
+Slug: 2014_10_18_20min
+Date: 2014-10_18
+Lang: de
```

Comment:
This affects the language in which the content is displayed, right? So if we load the english version of the ccc-ch.ch website, it's not shown?

Can we get rid of this somehow in order to show the item in every language?